lib/vanagon/component.rb

```diff
@@ -391,5 +392,13 @@ def environment_variables
     def rules(project, platform)
       Vanagon::Component::Rules.new(self, project, platform)
     end
+
+    # Converts the component to a Dockerfile that contains the
+    # build instructions.
+    #
+    # @return [String] the contents of the component's Dockerfile
+    def to_dockerfile(project, platform)
```

Comment:
Should this just be `dockerfile`, to match the name above?

Reply:
+1